tests/test_date.py

```diff
@@ -101,8 +101,9 @@ def test_get_date_custom_regex():
     """
     A valid regex with a matching filename. Returns a datetime.
     """
-    date_regex = re.compile(r"(?P<day>\d{2})\.(?P<month>\d{2})\.(?P<year>\d{4})[_-]?(?P<hour>\d{2})\.(?P<minute>\d{2})\.(?P<second>\d{2})")
-    assert Date("IMG_27.01.2015-19.20.00.jpg").from_exif({}, False, date_regex) == {
+    date_regex = re.compile(r"(?P<day>\d{2})\.(?P<month>\d{2})\.(?P<year>\d{4})[_-]?(?P<hour>\d{2})\.(?P<minute>\d{2})\.(?P<second>\d{2})")     # noqa: E501
```

Comment:
What's #noqa: E501

Reply:
If code is checked for PEP8 you can exclude a rule, in this case rule E501. PEP8 lint tools will skip a line with e.g. #noqa: E501.
In this case I made a decision that splitting this line would be counter-productive. To keep it in one line is crossing the 79 characters per line requirement from PEP8 therefor the #noqa: E501.